src/TestPlasterManifest.ps1

```diff
@@ -14,7 +14,7 @@ function Test-PlasterManifest {
     )
 
     begin {
-        $schemaPath = "$PSScriptRoot\Schema\PlasterManifest-v1.xsd"
+        $schemaPath = [io.path]::combine($PSScriptRoot, "Schema", "PlasterManifest-v1.xsd")
```

Comment:
Good catch and thanks for testing this on a non-Windows platform. Would you mind changing this slightly to:
```
... = [System.IO.Path]::Combine(...
```